core/templates/pages/library-page/library-page.component.ts

```diff
@@ -294,7 +294,8 @@ angular.module('oppia').component('libraryPage', {
                           ctrl.activitiesOwned.collections[
                             activitySummaryDict.id] = false;
                         } else {
-                          $log.error('INVALID ACTIVITY TYPE: Activity' +
+                          $log.error(
+                            'INVALID ACTIVITY TYPE: Activity' +
                           '(id: ' + activitySummaryDict.id +
```

Comment:
The indentation look weird here!

Reply:
Why the lint didn't catch this issue? (Just wanted to check!)

Reply:
This is a string, eslint allow these types of indentation for strings.